src/vlogger/vlogger.h

```diff
@@ -319,6 +325,9 @@ static inline void vlog_print_buffer(vlog_levels_t log_level, const char* msg_he
 		len = snprintf(buf, sizeof(buf)-1, "%s %s: ",
 			       g_vlogger_module_name, log_level::to_str(log_level));
 	}
+	if (len < 0) {
+		return ;
+	}
 	buf[len+1] = '\0';
 
 
```

Comment:
@OphirMunk I did not consider changing all place I see code is not good enough in this pull request. I am focusing on issues reported by clang only. There are other places in code that do not validate return value for snprintf.